prov/sm2/src/sm2_util.c

```diff
@@ -53,9 +52,8 @@ void sm2_cleanup(void)
 	struct dlist_entry *tmp;
 
 	pthread_mutex_lock(&sm2_ep_list_lock);
-	dlist_foreach_container_safe(&sm2_ep_name_list, struct sm2_ep_name,
-				     ep_name, entry, tmp)
-		free(ep_name);
+	dlist_foreach_container_safe(&sm2_ep_name_list, struct sm2_ep_name, ep_name,
+				     entry, tmp) free(ep_name);
```

Comment:
Formatting wrong

Reply:
We probably need to specify our macros similarly to the linux kernel clang-format file https://github.com/torvalds/linux/blob/master/.clang-format#L71

Reply:
done